src/vumps_localham.jl

```diff
@@ -297,15 +371,21 @@ function vumps_iteration_sequential(
       return noprime(UAC) * noprime(dag(UC))
     end
 
-    Ãᴸ[n] = ortho_polar(Ãᶜ[n], C̃[n])
-    Ãᴿ[n] = ortho_polar(Ãᶜ[n], C̃[n - 1])
+    function ortho_polar_right(AC, C)
+      UAC, _ = polar(dag(AC), uniqueinds(AC, C))
+      UC, _ = polar(dag(C), commoninds(C, AC))
+      return noprime(dag(UAC)) * noprime(UC)
+    end
 
+    Ãᴸ[n] = ortho_polar(Ãᶜ[n], C̃[n])
+    Ãᴿ[n] = ortho_polar_right(Ãᶜ[n], C̃[n - 1])
```

Comment:
Did the previous code cause a problem that you saw? I preferred trying to just have one function, but I don't think I had convinced myself that the previous code was properly equivalent to a right polar decomposition. 

The previous VUMPS code is working to very high accuracy in practice so I presume the previous version was correct.

Reply:
I will check that tomorrow again, I think your code relied on the fact that you had two site per unit cell + some real terms.
I might be wrong though.

I agree that having one function would be best.

Reply:
You were right, the previous code was working. Got confused by the indices.